### ISSUE TITLE
New folder Media Manager Page [Backend - side] Using Tool  [A11y]

### ISSUE THREAD

**Issue by adj9** [joomla/joomla-cms#31519]
### Description of the issue
Use Tool: [ARC Toolkit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce) on Chrome.
Test on new folder -  Media Manager Page.

### Error Results
<b> Keyboard - category</b>
1. <b>Tabindex sub-category</b>

- 3 errors with description - code problem on one HTML line type

Description:

> Error: tabindex is used on a non-interactive component in a way that puts it in the tab order. Only active element should be in the tab order
  Recommend: Remove the tabindex attribute or set it to '-1', OR define the role of the control.

These errors from the tool are classified as 
<b>non Active Element In Tab Order</b>.

Line of code error detect: `<div tabindex="0" class=""></div>`

2. <b>Tab order sub-category</b>
- 1 error

It is the tracker of eye movement on the entire page. There is'n a description.

1. <b> ARIA - category - ARIA usage sub-category</b>
- 5 errors with description - code problem on one HTML line type

Description:

> Error: aria-label is used on an element which does not reliably convey this attribute
  Recommend: Put aria-label on an active element or an element with a role defined, or use visually hidden text instead

<b> aria Label On Non Interactive Element</b>

Lines of code error detect:
- `<span aria-label="Toggle select item" title="Toggle select item" class="media-browser-select"></span>`

**Comment by ghost** [comment]
The issue should get the label `a11y`.
